### PR TITLE
NOISSUE - Improve explanation on message types for export service

### PR DESCRIPTION
**Pull request by mteodor**
### What does this do?
 improve explanation on message types for export service

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

File: docs/edge.md [hunk 1]
```diff
@@ -321,8 +323,8 @@ To match Mainflux requirements `mqtt_topic` must contain `channel/<channel_id>/m
...
+    - `default` is for sending messages as they are received on NATS with no transformation (so they should be in SenML or JSON format if we want to persist them in Mainflux in cloud). If you don't want to persist messages in Mainflux or you are not exporting to Mainflux cloud - message format can be anything that suits your application as message passes untransformed.
+    - `mfx` is for messages that are being picked up on internal Mainflux NATS bus. When using `Export` along with Mainflux deployed on gateway ([Fig. 1](#edge)) messages coming from MQTT broker that are published to NATS bus are [Mainflux message](https://github.com/mainflux/mainflux/blob/master/pkg/messaging/message.proto). Using `mfx` type will extract payload and `export` will publish it to `mqtt_topic`. Extracted payload is SenML or JSON if we want to persist messages. `nats_topic` in this case must be `channels`, or if you want to pick messages from a specific channel in local Mainflux instance to be exported to cloud you can put `channels.<local_mainflux_channel_id>`.
```
Comment (drasko) [inline docs/edge.md]: Probably add this link for proto to a footnote, as other are. For consistency sake.

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by drasko** [comment]
@mteodor please fix demanded and let's move forward with this one

File: docs/edge.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ Services that can be used on gateway to enable data and control plane for edge:
 * [Export](/edge/#export)
 * [Mainflux](/architecture/)
 
-![Edge](img/edge/edge.png)
+![Edge1](img/edge/edge.png)
+*Fig. 1*
```
Comment (drasko) [inline docs/edge.md]: Center this, also write: *Figure 1* - Description of the figure

File: docs/edge.md [hunk 3]
```diff
@@ -9,7 +9,9 @@ Services that can be used on gateway to enable data and control plane for edge:
...
+
 
 Figure shows edge gateway that is running Agent, Export and minimal deployment of Mainflux services.
```
Comment (drasko) [inline docs/edge.md:17]: Re there empty spaces in the front or I am not seeing well in GitHub?

File: docs/edge.md [hunk 4]
```diff
@@ -9,7 +9,9 @@ Services that can be used on gateway to enable data and control plane for edge:
 * [Export](/edge/#export)
 * [Mainflux](/architecture/)
 
-![Edge](img/edge/edge.png)
+![Edge1](img/edge/edge.png)
+*Fig. 1*
+
```
Comment (drasko) [inline docs/edge.md:15]: Why this 2 lines skip - would only one line skip work?

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**